mysql-test/suite/rocksdb/t/optimize_myrocks_replace_into_base.test

```diff
@@ -0,0 +1,79 @@
+--source include/have_rocksdb.inc
+SET @prior_rocksdb_perf_context_level = @@rocksdb_perf_context_level;
+SET GLOBAL rocksdb_perf_context_level=3;
+
+#
+# cast 1: table only with primary key, support replace blind write
```

Comment:
These test cases are great. Can they be extended to master/slave setups too to verify the slaves? But we'd also want to test combinations where the master has triggers/secondary keys, the slave doesn't, or where the slave has triggers/secondary keys, but the master does not.

The reason these cases are important is because schema changes and triggers are applied to each member of a replicaset at different times, so there are times when schemas between master and slave will different.